### PR TITLE
feature - adds govau header and footer; app_shell; fe ops build;

### PR DESCRIPTION
**Pull request by elisechant**
* adds a gov au header
* adds a gov au footer 
* update build pipeline which changes the way assets are hashed and cached by browsers

### CODE REVIEW

**Comment by elisechant** [comment]
http://performance-dashboard-ui.surge.sh/?knob-text=Bacon%20ipsum%20lorem&selectedKind=Ui-kit%20GOV.AU%20Header&selectedStory=Default&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel

screen shot 2016-12-02 at 6 38 42 pm

File: client/src/_shared/styles/_vendor/bootstrap/uikit-components/_govau-header.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        display: block;
+        margin-bottom: 8px;
+      }
+
+      .user-email {
+        position: relative;
+
+        svg {
+          position: absolute;
+          left: -26px;
```
Comment (simonschwartz) [inline client/src/_shared/styles/_vendor/bootstrap/uikit-components/_govau-header.scss]: Does this still achieve what you intended if you remove the `position: absolute;` and replace `left: -26px;` with `margin-right: <spacing here>`?

Comment (elisechant) [inline client/src/_shared/styles/_vendor/bootstrap/uikit-components/_govau-header.scss]: @simonschwartz good idea

**Comment by elisechant** [comment]
todo: 

* login.css pollutes the global scope, so button and links in the footer are getting clobbered 🎉  - fix
* browser verify 
* mobile optimisations
* verify cached assets work as expected

**Pull request merged**